### PR TITLE
Sort existing use statements alphabetically

### PR DESCRIPTION
**Pull request by AlexMasterov**

### CODE REVIEW

File: tests/Formatter/PlatesFormatterTest.php [hunk 1]
```diff
@@ -6,8 +6,9 @@
 use Equip\Payload;
 use League\Plates\Engine;
 use Lukasoppermann\Httpstatus\Httpstatus;
+use PHPUnit_Framework_TestCase as TestCase;
 
-class PlatesFormatterTest extends \PHPUnit_Framework_TestCase
+class PlatesFormatterTest extends TestCase
```
Comment (shadowhand) [inline tests/Formatter/PlatesFormatterTest.php]: 👍

File: tests/Handler/ActionHandlerTest.php [hunk 2]
```diff
@@ -1,11 +1,11 @@
 <?php
 namespace EquipTests\Handler;
 
+use EquipTests\Configuration\ConfigurationTestCase;
+use EquipTests\Fake\FakeDomain;
```
Comment (shadowhand) [inline tests/Handler/ActionHandlerTest.php]: Actually these were in the right place. `Equip\` sorts lower than `EquipTests\`.

Comment (AlexMasterov) [inline tests/Handler/ActionHandlerTest.php]: Okay. Returned as it was.

File: tests/Formatter/PlatesFormatterTest.php (hunk 1 above)
Comment (AlexMasterov) [inline tests/Formatter/PlatesFormatterTest.php]: Oh. I need to create a separate pull request for this?

Comment (shadowhand) [inline tests/Formatter/PlatesFormatterTest.php]: Nope.

**Comment by shadowhand** [comment]
👍

**Pull request merged**